code/__HELPERS/sanitize_values.dm

```diff
@@ -16,6 +16,9 @@
 	if(default)			return default
 	if(List && List.len)return List[1]
 
+/proc/sanitize_islist(value, default)
+	if(value && length(value) && islist(value))	return value
```

Comment:
Just length(value) is enough (no need to double check, it if it's null length wont' pass)
Also, single line control statement

Reply:
Following the convention of the file here (with the single line control statement)